### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by II11II** [jhomlala/alice#84]
```
What went wrong:
Execution failed for task ':app:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class com.google.android.exoplayer2.ui.package-info found in modules jetified-exoplayer-common-2.14.1-runtime (com.google.android.exoplayer:exoplayer-common:2.14.1) and jetified-exoplayer-ui-2.13.3-runtime (com.google.android.exoplayer:exoplayer-ui:2.13.3)

     Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.

* Try:
Run with --stack trace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
```
I added ```alice: ^0.2.2```

And it gives an error

``` Flutter version: 2.2.2 ```

**My app dependencies:**
```

  flutter_bloc: ^7.0.0
  hydrated_bloc: ^7.0.0
  uuid: ^3.0.4
  mask_text_input_formatter: ^2.0.0-nullsafety.2
  url_launcher: ^6.0.3
  easy_localization: ^3.0.0
  path_provider: ^2.0.2
  get_it: ^7.1.3
  geolocator: ^7.0.1
  rxdart: ^0.27.1
  
  path: ^1.8.0
  google_maps_flutter: ^2.0.3
  firebase_messaging: ^10.0.1
  http_interceptor: ^0.4.1
  http: ^0.13.3

  logger: ^1.0.0-nullsafety.0
  equatable: ^2.0.0
  cupertino_icons: ^1.0.2
  freezed_annotation:
  lottie: ^1.0.1
  cached_network_image: ^3.0.0-nullsafety
  video_thumbnail: ^0.3.3
  camera: ^0.8.1+3
  # camera: 
  #   git:
  #     url: https://github.com/bottlepay/plugins.git
  #     path: packages/camera/camera
  #     ref: android-rework

  flutter_svg: ^0.22.0
  flutter_svg_provider: ^0.1.9
  device_info: ^2.0.0
  package_info: ^2.0.2
  pin_code_fields: ^7.0.0
  smooth_page_indicator: ^0.3.0-nullsafety.0
  dotted_border: ^2.0.0-nullsafety.0
  flutter_google_places: ^0.3.0
  geocoding: ^2.0.0
  photo_manager: ^1.1.0
  image_picker: ^0.7.4

  story_view: ^0.13.1
  
  video_player: ^2.1.6

  carousel_slider: ^4.0.0-nullsafety.0
  flutter_staggered_animations: ^1.0.0
  shimmer: ^2.0.0
  custom_sliding_segmented_control: ^1.0.0
  flutter_rating_bar: ^4.0.0
  table_calendar: ^3.0.0
  grouped_list: ^4.0.0
  
dependency_overrides:
    rxdart: ^0.27.1
```

**Comment by II11II** [comment]
After forking and removing better_player dependency it worked

**Comment by jhomlala** [comment]
Fixed in 0.2.3.
